interfaces/api/v1/audit/review_report_routes.py

```
@@ -0,0 +1,58 @@
+                    description=issue.description,
+                    location=issue.location,
+                    suggestion=issue.suggestion,
+                )
+                for issue in report.issues
+            ],
+            overall_score=report.overall_score,
+            summary=report.summary,
+            reviewed_at=report.reviewed_at.isoformat(),
+        )
+    except ValueError as e:
+        raise HTTPException(status_code=404, detail=str(e))
```

Ruff (0.15.10)

[warning] Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)